### DISCUSSION TITLE
Please edit rpcUrl for 'Klaytn' network.

### DISCUSSION THREAD

**Discussion by judeProground** [wevm/viem#1630] [Question]
Hello!
Please edit the rpcUrl `https://klaytn.drpc.org/`(which gives HTTP request failed 500 errors) for the 'Klaytn' network in [chain definitions](https://github.com/wevm/viem/blob/3637d927b8e1f1788771f3e8c7c85f13f5171111/src/chains/definitions/klaytn.ts#L13C15-L13C15)

```
Status: 500
URL: https://klaytn.drpc.org
Request body: {"method":"eth_estimateGas","params":[{"from":"0xf58d1Dd6649fD98867FB1faAc4a80c02ec0b577e","to":"0x8da74ba3db7b440264421373d2cbfca2b10e2e3b","value":"0x38d7ea4c68000"}]}

Estimate Gas Arguments:
  from:   0xf58d1Dd6649fD98867FB1faAc4a80c02ec0b577e
  to:     0x8da74ba3db7b440264421373d2cbfca2b10e2e3b
  value:  0.001 KLAY

Details: {"message":"Unable to perform request","code":19}
Version: viem@1.19.12
```

According to the [official Klaytn documentation](https://docs.klaytn.foundation/content/dapp/tutorials/connecting-metamask#connect-to-klaytn-cypress-network-mainnet), it is recommended to use `https://public-en-cypress.klaytn.net` as the RPC URL. This change should resolve the issues we're experiencing with transaction signing and other network-related functions.

Or, if it's alright, may I submit a PR (Pull Request) myself?

**Comment by tmm** [comment]
PR welcome
